canonical-web-and-design/canonical.com
# Change job application form on /careers/<department>

## Done
- Change job application form on `/careers/<department>`


## QA
- Run the website locally using `./run`
- Go to [/careers/sales](http://0.0.0.0:8002/careers/sales) and make sure you can make an application.


## Issue
Fixes https://github.com/canonical-web-and-design/web-squad/issues/1836
